### PR TITLE
Toast

### PR DESCRIPTION
**Pull request by Shubhamkr2610**
Toast component
I have added the toast component in component library. So ignore other files. Focus only on the toast.html ,  toast.css (in toast folder ) , (in components) .

What not to check
Responsiveness

image

### CODE REVIEW

**Review by tonymike99 — APPROVED** [review]
Hi Shubham,
The toast looks really nice. Nice code.
Good to merge!

**Review by sumitk-singh — APPROVED** [review]
codes looks clean, good to merge.

**Comment by devAmlan** [comment]
Everything looks good. At line no. 75 u can use rem instead of px.

File: components/toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+.btn {
+    font-size: 1.25rem;
+    cursor: pointer;
+    border: none;
+    outline: none;
+    background-color: transparent;
+}
+
+.fa {
```
Comment (iamalhera) [inline components/toast/toast.css:74]: make another class of the icon tag  to make it specifically

**Pull request merged**